samples/tfm_integration/tfm_ipc/README.rst

```diff
@@ -0,0 +1,84 @@
+.. _tfm_ipc:
+
+TF-M IPC
+########
+
+Overview
+********
+This is a simple TF-M integration example that can be used with an Arm v8-M
+supported board.
+Zephyr uses Trusted Firmware (TF-M) Platform Security Architecture (PSA) APIs
+to run this sample in a secure configuration with Zephyr itself in a
+non-secure configuration.
+The sample prints test info to the console either as a single-thread or
+multi-thread application.
+
+Building and Running
+********************
+
+This project outputs test status and info to the console. It can be built and
+executed on MPS2+ AN521.
+
+On MPS2+ AN521:
+===============
+
+1. Build Zephyr with a non-secure configuration (``-DBOARD=mps2_an521_nonsecure``).
+
+   .. code-block:: bash
+
+      cd $ZEPHYR_ROOT/samples/tfm_integration/tfm_ipc/
+      mkdir build
+      cd build
+      cmake -GNinja -DBOARD=mps2_an521_nonsecure ..
+      ninja -v
+
+2. Copy application binary files (mcuboot.bin and tfm_sign.bin) to ``<MPS2 device name>/SOFTWARE/``.
+3. Open ``<MPS2 device name>/MB/HBI0263C/AN521/images.txt``.
```

Comment:
These two steps should be combined to say:
```
#. Edit ``<MPS2 device name>/MB/HBI0263C/AN521/images.txt`` as follows:
```

You can use ``#.`` for auto-numbering, instead of a hard numbered step.